### ISSUE TITLE
Strobe lights in yeti's revenge final castle

### ISSUE THREAD

**Issue by Frostwithasideofsalt** [SuperTux/addons#48]
in the final castle, the background is a solid color that changes INSANELY fast. there is no doubt in my mind that if someone with epilepsy saw this, they would have a seizure . this addon should either be removed from the repository or someone should fix that, urgently.

**Comment by Vankata453** [comment]
You can fix it if you want to. Don't think the original creator is around.

**Comment by Frostwithasideofsalt** [comment]
> The problem is that the strobe lights are a gameplay element as their color change indicates the right way.

okay but uhh counterpoint strobe lights. just communicate the thing some other way that couldn't result in someone dying

**Comment by tylerandari13** [comment]
> The problem is that the strobe lights are a gameplay element as their color change indicates the right way.

This is false. I played the addon and its a big puzzle room that has nothing to do with lights. The strobe lights are probably there to make it seem magical.

**Comment by tobbi** [comment]
> > The problem is that the strobe lights are a gameplay element as their color change indicates the right way.
> 
> This is false. I played the addon and its a big puzzle room that has nothing to do with lights. The strobe lights are probably there to make it seem magical.

I thought when you did something right it flashed slightly differently? Don't you have to carry certain blocks for something to happen?

**Comment by tylerandari13** [comment]
> > > The problem is that the strobe lights are a gameplay element as their color change indicates the right way.
> > 
> > 
> > This is false. I played the addon and its a big puzzle room that has nothing to do with lights. The strobe lights are probably there to make it seem magical.
> 
> I thought when you did something right it flashed slightly differently? Don't you have to carry certain blocks for something to happen?

I believe the strobe lights get faster, but this is only a visual thing and proves Frosts point even more.
